model-metadata/CU-ensemble.yml

```
@@ -1,35 +1,40 @@
 team_name: "Columbia University"
 team_abbr: "CU"
-model_name: "Ensemble"
-model_abbr: "ensemble"
+model_name: "Columbia University Ensemble"
+model_abbr: "CU-ensemble"
```

This will fix the model_abbr error

```suggestion
model_abbr: "CU_ensemble"
```